packages/file/pubspec.yaml

```diff
@@ -7,10 +7,10 @@ description:
 repository: https://github.com/google/file.dart/tree/master/packages/file
 
 environment:
-  sdk: '>=3.0.0-0 <4.0.0'
+  sdk: '>=3.0.0-356.0.dev <4.0.0'
```

Comment:
Almost entirely academic, but this could be as low as `>=3.0.0-321.0.dev`. Realistically it isn't going to matter much so if you don't feel like updating it, don't :).

Reply:
On second thought, `>=3.0.0-325.0.dev` is probably better because that will also enable patterns/records which technically could break some things (although we know they don't already, at least for this package).

Reply:
For me, I would just let 3.0.0-0 and merge, but let's go.. lol

Reply:
I have a random question, I'm not following Dart that closely, but can I already do `switch(a) => option1 => 1, option2 => 2`?

Reply:
I think it looks a bit different than that but yes